### DISCUSSION TITLE
useRadioGroup onChange not triggered

### DISCUSSION THREAD

**Discussion by EhanDuan** [chakra-ui/chakra-ui#6240] [Q&A]
Hi, I met some trouble when using **a custom radio group.**

## Official Example
 I am looking at `radio` example here. https://chakra-ui.com/docs/components/radio/usage
<img width="792" alt="image" src="https://user-images.githubusercontent.com/75397936/175625767-61476ce8-8cb4-4869-a35c-9145733ee6b6.png">

Everything goes well on this page. I inspect and click item, there would be relative result in the console.

## My code
I extracted the `RadioCard` in `radioCard.tsx` under component folder. The code was exactly the same as the example.
```tsx
import { Box, useRadio } from "@chakra-ui/react";

function RadioCard(props: any) {
  const { getInputProps, getCheckboxProps } = useRadio(props);

  const input = getInputProps();
  const checkbox = getCheckboxProps();

  return (
    <Box as="label">
      <input {...input} />
      <Box
        {...checkbox}
        cursor="pointer"
        borderWidth="1px"
        borderRadius="md"
        boxShadow="md"
        _checked={{
          bg: "teal.600",
          color: "white",
          borderColor: "teal.600"
        }}
        _focus={{
          boxShadow: "outline"
        }}
        px={5}
        py={3}
      >
        {props.children}
      </Box>
    </Box>
  );
}

export default RadioCard;

```
I imported the card into a page, the code was also the same as example one. **But when I clicked items, there was no result show in the console.** 
```tsx
const ProblemUpload = () => {
  const options = ["react", "vue", "svelte"];

  const { getRootProps, getRadioProps } = useRadioGroup({
    name: "framework",
    defaultValue: "react",
    onChange: console.log
  });

  const group = getRootProps();

  return (
    <HStack {...group}>
      {options.map((value) => {
        const radio = getRadioProps({ value });
        return (
          <RadioCard key={value} {...radio}>
            {value}
          </RadioCard>
        );
      })}
    </HStack>
  );
};
```

## Thought
+ Is it due to I **separated possibly one file into two files in some wrong manner**?
+ In addition, I put the example code on a single page. It also did not show the result in the console. So the reason might not be the first one. Do you know why ? 

Thanks for your help!

**Comment by jmoldenbushel** [comment]
Having the same issue.

**Comment by ThaNico** [comment]
I've had the same issue, I printed it this way :
`onChange: (newValue) => console.log(newValue)`
